### PR TITLE
feat(swiper): remove custom swipe controls, implement swiper

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

**Which issue(s) this PR fixes** 🔨

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: layouts/server.vue [hunk 1]
```diff
@@ -54,17 +67,52 @@ import { mapState } from 'vuex'
...
   },
   computed: {
     ...mapState(['ui', 'media']),
+    swiper() {
+      return this.$refs.swiper.$swiper
+    },
+  },
+  methods: {
+    toggleMenu() {
+      console.log('toggleMenu')
```
Comment (Jekrimo) [inline layouts/server.vue:109]: console.log here

**Review by Jekrimo — DISMISSED** [review]

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
Great job! Can you make the following small changes:

1. Remove the console.log messages @Jekrimo mentioned, throughout. If you want to leave logging in please use `this.$Logger`
2. In the layout files where you load in Swiper/SwiperSlide, can you remove the imports (js/css), and remove them from the components block. Can you:

- Create a file at `plugins/thirdparty/swiper.ts`, something like
```
import Vue from 'vue'
import { Swiper, SwiperSlide } from 'vue-awesome-swiper'
import 'swiper/css/swiper.css'

Vue.component('Swiper', Swiper)
Vue.component('SwiperSlide', SwiperSlide)
```

and import it into the nuxt.config.js plugins?

File: components/views/navigation/sidebar/Sidebar.vue [hunk 2]
```diff
@@ -58,15 +66,18 @@ export default Vue.extend({
      * when user is on a mobile device
      */
     if (this.$route.name?.includes('chat-direct') && this.$device.isMobile) {
-      this.$props.toggle()
+      // this.$props.toggle()
```
Comment (RetroPronghorn) [inline components/views/navigation/sidebar/Sidebar.vue:69]: let's delete commented out code in this file.

File: components/views/navigation/sidebar/Sidebar.vue [hunk 3]
```diff
@@ -58,15 +66,18 @@ export default Vue.extend({
...
+      this.$props.showMenu()
     }
 
     if (this.$route.name?.includes('friends-list') && this.$device.isMobile) {
-      this.$props.toggle()
+      // this.$props.toggle()
```
Comment (RetroPronghorn) [inline components/views/navigation/sidebar/Sidebar.vue:74]: delete

File: components/views/navigation/sidebar/Sidebar.vue [hunk 4]
```diff
@@ -58,15 +66,18 @@ export default Vue.extend({
...
+      this.$props.showMenu()
     }
 
     if (this.$route.name?.includes('files-browse') && this.$device.isMobile) {
-      this.$props.toggle()
+      // this.$props.toggle()
```
Comment (RetroPronghorn) [inline components/views/navigation/sidebar/Sidebar.vue:79]: delete

File: components/views/settings/modal/Modal.less [hunk 5]
```diff
@@ -17,9 +17,24 @@ p {
...
+  
+  .not-sidebar {
+    width: 100%;
+  }
+}
+
 .sidebar {
-  flex-basis: 18rem;
-  flex-grow: 1;
+  // flex-basis: 18rem;
```
Comment (RetroPronghorn) [inline components/views/settings/modal/Modal.less:36]: delete

File: components/views/settings/modal/Modal.less [hunk 6]
```diff
@@ -35,43 +50,75 @@ p {
     opacity: 0;
   }
 
-  .is-collapsed {
-    .menu-toggle {
-      right: 1rem !important;
-    }
-  }
+  // .is-collapsed {
```
Comment (RetroPronghorn) [inline components/views/settings/modal/Modal.less:53]: delete

File: layouts/Layout.less [hunk 7]
```diff
@@ -93,16 +123,16 @@
...
       width: calc(@full - 2rem) !important;
     }
-    #sidebar,
-    #servers-vertical-list {
-      transform: translateX(calc(-@sidebar-size + -@servers-size));
-    }
-    .dynamic-content {
-      margin-left: calc(-@sidebar-size + -@servers-size);
-    }
+  //   #sidebar,
```
Comment (RetroPronghorn) [inline layouts/Layout.less:126]: delete

File: libraries/Textile/TextileManager.ts [hunk 8]
```diff
@@ -47,7 +47,7 @@ export default class TextileManager {
 
     this.mailboxManager = new MailboxManager(textile, wallet.address)
     this.bucketManager = new BucketManager(textile, identity, textile.wallet.address)
-    await this.bucketManager.init()
+    // await this.bucketManager.init().catch((e) => console.log(e))
```
Comment (RetroPronghorn) [inline libraries/Textile/TextileManager.ts:50]: Please un comment. @stavares843 note this must me commented out to test on mobile devices.

File: libraries/Textile/TextileManager.ts [hunk 9]
```diff
@@ -70,7 +70,7 @@ export default class TextileManager {
     this.mailboxManager = new MailboxManager(textile, textile.wallet.address)
     await this.mailboxManager.init()
     this.bucketManager = new BucketManager(textile, textile.identity,textile.wallet.address)
-    await this.bucketManager.init()
+    // await this.bucketManager.init().catch((e) => console.log(e))
```
Comment (RetroPronghorn) [inline libraries/Textile/TextileManager.ts:73]: Please un comment. @stavares843 note this must me commented out to test on mobile devices.

**Review by RetroPronghorn — CHANGES_REQUESTED** [review]

**Review by RetroPronghorn — APPROVED** [review]

File: libraries/Textile/TextileManager.ts (hunk 9 above)
Comment (stavares843) [reply libraries/Textile/TextileManager.ts]: noted, thanks! 💥

**Review by WanderingHogan — APPROVED** [review]

**Comment by WanderingHogan** [comment]
This is good to go outside of the conflict. Thank you for making the changes.

**Comment by WanderingHogan** [comment]
* if it passes QA

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
this needs to be improved performance-wise but will add a ticket for that to be improved

**Pull request merged**